jwst/dark_current/dark_current_step.py

```diff
@@ -86,19 +86,19 @@ def set_average_dark_current(self, input_model, dark_model):
         Take the three possible locations specifying
         the average dark current and assign them to the
         input model, in priority order:
-        1) Any value provided to the step parameter, either from
-        the user or a parameter reference file
-        2) The 2-D array stored in dark_model.average_dark_current
-        3) The scalar value stored in dark_model.meta.exposure.average_dark_current
+
+        1. Any value provided to the step parameter, either from
+           the user or a parameter reference file
+        2. The 2-D array stored in dark_model.average_dark_current
+        3. The scalar value stored in dark_model.meta.exposure.average_dark_current
 
         Parameters
         ----------
         input_model : `~stdatamodels.jwst.datamodels.RampModel`
             The input datamodel containing the 4-D ramp array.
-        dark_model : Union[stdatamodels.jwst.datamodels.DarkModel,
-                           stdatamodels.jwst.datamodels.DarkMIRIModel]
+        dark_model : `~stdatamodels.jwst.datamodels.DarkModel` or `~stdatamodels.jwst.datamodels.DarkMIRIModel`
```

Comment:
Instead of adding `noqa` here, you can add a backslash at the end of the line for a long type specification.

```suggestion
        dark_model : `~stdatamodels.jwst.datamodels.DarkModel` or \
                     `~stdatamodels.jwst.datamodels.DarkMIRIModel`
```

Reply:
Should be!  I tested it locally too.

Reply:
Ah ok. One more thing, since `\` is "discouraged" in code... Would you rather we do this to keep line length without the `\` thingy?

https://github.com/astropy/astropy/blob/6b9d2f7b7b91efaa65a94348c33c6d0ef8a02f3a/docs/conf.py#L447-L453

Reply:
Oh, that looks really useful!  I didn't know you could do that. I dislike having to type out all the references every time, so something like that would be great to help people actually write useful references.  Maybe in another PR, though?

Reply:
Hmm nvm. Now that I dig, it is for narrative doc, not numpydoc:

* https://github.com/astropy/astropy/pull/16162